Sources/Baggage/BaggageContext.swift

```diff
@@ -159,3 +162,79 @@ extension AnyBaggageContextKey: Hashable {
+    /// This function is really only intended to be used frameworks and libraries, at the "top-level" where a request's,
+    /// message's or task's processing is initiated. For example, a framework handling requests, should create an empty
+    /// context when handling a request only to immediately populate it with useful trace information extracted from e.g.
+    /// request headers.
+    ///
+    /// ### Usage in applications
+    /// More often than not application code should never have to create an empty context.
+    ///
+    /// Usually, a framework such as an HTTP server or similar "request handler" would already provide users
+    /// with a context to be passed along through subsequent calls.
+    ///
+    /// If unsure where to obtain a context from, prefer using `TODO("Not sure where I should get a context from here?")`,
+    /// such that other developers are informed that the lack of context was not done on purpose, but rather because either
+    /// not being sure where to obtain a context from, or other framework limitations -- e.g. the outer framework not being
+    /// context aware just yet.
+    public static var empty: BaggageContext {
+        return BaggageContext()
+    }
+}
+
+// ==== ----------------------------------------------------------------------------------------------------------------
+// MARK: "TO DO" BaggageContext
+
+extension BaggageContextProtocol {
+    /// A baggage context intended as a placeholder until a real value can be passed through a function call.
+    ///
+    /// It should ONLY be used while prototyping or when the passing of the proper context is not yet possible,
+    /// e.g. because an external library did not pass it correctly and has to be fixed before the proper context
+    /// can be obtained where the TO-DO is currently used.
+    ///
+    /// ### Crashing on TO-DO context creation
+    /// You may set the `BAGGAGE_CRASH_TODOS` variable while compiling a project in order to make calls to this function crash
+    /// with a fatal error, indicating where a to-do baggage context was used. This comes in handy when wanting to ensure that
+    /// a project never ends up using with code which initially was written as "was lazy, did not pass context", yet the
+    /// project requires context passing to be done correctly throughout the application. Similar checks can be performed
+    /// at compile time easily using linters (not yet implemented), since it is always valid enough to detect a to-do context
+    /// being passed as illegal and warn or error when spotted.
+    ///
+    /// - Parameters:
+    ///   - reason: Informational reason for developers, why a placeholder context was used instead of a proper one,
+    /// - Returns: Empty "to-do" baggage context which should be eventually replaced with a carried through one, or `background`.
+    public static func TODO(_ reason: String, function: String = #function, file: String = #file, line: UInt = #line) -> BaggageContext {
```

Comment:
```suggestion
    public static func TODO(_ reason: StaticString, function: String = #function, file: String = #file, line: UInt = #line) -> BaggageContext {
```

Nothing major, but through having the `reason` be a `StaticString` we might express more clearly that this should be something like a comment instead of the result of some crazy String interpolation.

Reply:
Yeah might as well here agreed